### PR TITLE
Adds ENE guide

### PR DESCRIPTION
**Pull request by schalkventer**
image

image

image

image

image

### CODE REVIEW

**Comment by jbothma** [comment]
The guide list page heading and description don't really mean anything to me - nothing there indicates that I'm looking at a list of guides that tell me more about how to use the data and analyses on the site.

Also no tab is selected

**Comment by jbothma** [comment]
The thing on the side of the dataset page says "what is an ENE" but that's a bit weird because the guide is intended to help you use the data on the page you're looking at. Something like "Learn more about this data" would be more appropriate given the journey we're trying to take the user on, wouldn't it?

**Comment by longhotsummer** [comment]
> The introduction is really overwhelming - a full page of text. Is there anything we can do to make it less so? What will this guide give me? Why should I read it?

Let me see what I can do here.

> Is it easy to add a table of contents?

**Comment by longhotsummer** [comment]
Also breadcrumbs on the dataset guide pages are necessary, I think.

**Comment by longhotsummer** [comment]
I think the main heading of the page should explicitly indicate it's a dataset guide. eg: `Dataset Guide: Estimates of National Expenditure`

File: guides/performance-and-expenditure-reviews.md [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
 ---
 
 
+{% assign page_data = site.data.guides.performance-and-expenditure-reviews %}
+
+
 <h1 class="Page-mainHeading">Performance and Expenditure Reviews (PER)</h1>
+
 <div class="BasicContent u-marginBottom15">
   <div class="BasicContent-title">Introduction</div>
```
Comment (longhotsummer) [inline guides/performance-and-expenditure-reviews.md]: Please change Introduction to Overview

**Comment by longhotsummer** [comment]
@schalkventer I've remove some of the content in the Overview section, see suggested edits here: https://docs.google.com/document/d/1YPFS9sX1xeFMVwYXZummoAUv6oe6gILBtvQZo-gdFBM/edit?ts=5bc70112# -- this should make it a little bit shorter.

File: guides/estimates-of-national-expenditure.md [hunk 2]
```diff
@@ -0,0 +1,312 @@
...
+    <li>
+      Expenditure trends and estimates for significant spending items
+    </li>
+
+    <li>
+      Expenditure trends and estimates for goods and services (economic classification Level 2)
+    </li>
+
+    <li>
+      Expenditure trends and estimates for transfers and subsidies  (economic classification Level 3
```
Comment (longhotsummer) [inline guides/estimates-of-national-expenditure.md]: Missing `)` at end.

File: guides/estimates-of-national-expenditure.md [hunk 3]
```diff
@@ -0,0 +1,312 @@
...
+    <li>
+      Personnel numbers and their costs by salary level and programme
+    </li>
+
+    <li>
+      Departmental receipts (economic classification Level 3)
+    </li>
+
+    <li>
+      Expenditure trends and estimates for each programme, by sub-programme and economic classification (Level 2), and by transfers and subsidies (Level 3))
```
Comment (longhotsummer) [inline guides/estimates-of-national-expenditure.md]: Duplicate `)` at end.

**Pull request merged**